modules/k8s_eks_addons/templates/autoscaler_values.yaml

```diff
@@ -2,20 +2,20 @@ awsRegion: ${aws_region}
 
 autoDiscovery:
   clusterName: ${eks_cluster_id}
+  tags:
+    - k8s.io/cluster-autoscaler/enabled
+    - k8s.io/cluster-autoscaler/${infrastructurename}
+    - k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage

```

Comment:
I could confirm (during tests for #168), that the `resources/ephemeral-storage` is not required here. It needs to be removed from here!

Reply:
To address your first comment, it is currently possible to pass any addition to the values file via variable cluster_autoscaler_config.chart_values. This variable will be appended to the values file or override existing value.

Removed resources/ephemeral-storage.

Reply:
I tried that out 1 or 2 weeks ago with the previous autoscaler configuration and did not get it running without errors.

Since `templates/autoscaler_values.yaml` already contains a value for `autoDiscovery.tags`, specifying another `autoDiscovery.tags` via `cluster_autoscaler_config.chart_values` cannot just be appended to the values file with getting syntax errors!?

And replacing values would mean, that one needs to specify the default tags also in `cluster_autoscaler_config.chart_values`, which is not very helpful.

Or did you already tried out, that a merge of these two sources for `autoDiscovery.tags` really works?

Reply:
That is correct, one needs to specify both default values and additional values for ```autoDiscovery.tags```.  ```autoDiscovery.tags``` is a list, not a sub dictionary, therefore it wouldn't be merged but replaced.

Solution for this would come with some compromises. Additional field in ```cluster_autoscaler_config``` can be created for ```autoDiscovery.tags``` which then will be appended/updated (with some magic of ```concat``` function).

In my opinion specifying both default values and additional values in ```chart_values``` is still a cleaner solution for this case.

Reply:
Disagree.

Many helm charts provide variables like `extraLabels` (or similar) to extend default list values. The naming `extra` (or `additional`) makes it very clear for everyone.

The downside of not having such an option is, that on version upgrades the default lists might change and no one is aware of this and checks/updates own overrides. This is a very _unclean solution_ ... in my opinion.

@sgrdn What do you think about that?

Reply:
Otherwise ... since we/i just have discovered that there is no need to enhance this list currently, we could call this _premature optimization_. And one could raise the question, whether there really is some use case where additional tags are needed.

Reply:
I have probably worded it in a wrong way. By clear I have meant clear implementation.
But now that I have thought it over, there is always a posibility of combining ```values``` argument and  ```set``` block.

If you need this, we can implement it.

Reply:
Since we don't need it at the moment, it might be best to use development capacities for more important stuff.